Tasks/Xcode/xcode.ts

```diff
@@ -45,21 +45,26 @@ async function run() {
             }
         }
 
-        var sdk : string = tl.getInput('sdk', false);
-        var configuration : string  = tl.getInput('configuration', false);
-        var scheme : string = tl.getInput('scheme', false);
-        var useXcpretty : boolean = tl.getBoolInput('useXcpretty', false);
-        var xctoolReporter : string = tl.getInput('xctoolReporter', false);
-        var actions : string [] = tl.getDelimitedInput('actions', ' ', true);
-        var packageApp : boolean = tl.getBoolInput('packageApp', true);
-        var args : string = tl.getInput('args', false);
+        var isProject = false;
+        if (ws.trim().toLowerCase().endsWith('.xcodeproj')) {
+            isProject = true;
+        }
```

Comment:
ws shouldn't be null even if xcWorkspacePath is not specified. It will default to the root of the repository since it is a path input. But it is a good idea to add the extra null check.